### PR TITLE
Feature/122

### PR DESCRIPTION
**Pull request by fraincs**
Added an overview page to the components page. This is the first of more PR as this it establishes the logic and some basic UI. UI refinement need to happen at another time.

Preview -> https://deploy-preview-305--wl-hopper.netlify.app/components/getting-started/overview

### CODE REVIEW

**Comment by alexasselin008** [comment]
What is the URL, can you update the description of the PR to add it?

**Comment by franckgaudin** [comment]
> What is the URL, can you update the description of the PR to add it?

https://deploy-preview-305--wl-hopper.netlify.app/components/getting-started/overview

File: apps/docs/app/ui/components/overview/Overview.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+const Overview = () => {
+    // Get unique categories
+    const categories = Array.from(new Set(allComponents.map(component => component.category)));
+
+    return (
+        <div className="hd-component-overview-wrapper">
+            {categories.map(category => (
+                <div className="hd-component-overview-category" key={category}>
+                    <h2 className="hd-component-overview-category__title">{category}</h2>
```
Comment (franckgaudin) [inline apps/docs/app/ui/components/overview/Overview.tsx:15]: You should use the Title component 
https://github.com/gsoft-inc/wl-hopper/blob/main/apps/docs/components/title/Title.tsx

File: apps/docs/app/ui/components/overview/Overview.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+"use client";
```
Comment (franckgaudin) [inline apps/docs/app/ui/components/overview/Overview.tsx:1]: Can't this component be rendered on the server side?

File: apps/docs/app/ui/components/overview/Overview.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    const categories = Array.from(new Set(allComponents.map(component => component.category)));
+
+    return (
+        <div className="hd-component-overview-wrapper">
+            {categories.map(category => (
+                <div className="hd-component-overview-category" key={category}>
+                    <h2 className="hd-component-overview-category__title">{category}</h2>
+                    <div className="hd-component-overview">
+                        {allComponents.filter(component => component.category && component.category === category).map(component => (
+                            <a key={component._id} className="hd-component-overview__item" href={`/${component._raw.flattenedPath}`}>
```
Comment (franckgaudin) [inline apps/docs/app/ui/components/overview/Overview.tsx:18]: You can use next's link component 
https://nextjs.org/docs/app/api-reference/components/link

File: apps/docs/app/ui/components/overview/Overview.tsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+                                <div className="hd-component-overview__item-thumb">
+                                    <Image src={`https://place-hold.it/260x140?text=${component.title}&fontsize=20`} alt="thumb" width="260" height="140" className="hd-component-overview__item-img" />
+                                </div>
+                                <div className="hd-component-overview-item__caption">
+                                    <h3>{component.title}</h3>
```
Comment (franckgaudin) [inline apps/docs/app/ui/components/overview/Overview.tsx:23]: Same as h2 using the Title component

**Comment by franckgaudin** [comment]
There are several conflicts with the branch hand that need to be resolved locally

**Review by franckgaudin — APPROVED** [review]

**Pull request merged**